rubin_sim/maf/glance_dir.py

```diff
@@ -12,7 +12,7 @@
 import argparse
 
 
-if __name__ == "__main__":
+def glance_dir():
     """
```

Comment:
I think it is usually preferred style not to put a newline right after the opening triple-quote, but to put the summary on the same line as it.